lib/galaxy/datatypes/binary.py

```diff
@@ -4540,3 +4540,11 @@ def display_peek(self, dataset: DatasetProtocol) -> str:
             return dataset.peek
         except Exception:
             return f"Binary FITS file size ({nice_size(dataset.get_size())})"
+
+class Docx(Binary):
```

Comment:
Not necessary. Just refer to Binary in the datatypes xml.

Reply:
ok, thank you!